app/pipeline/chat_gpt_wrapper_pipeline.py

```
@@ -0,0 +1,81 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.prompts import (
+    ChatPromptTemplate,
+)
+from app.common.pyris_message import IrisMessageRole, PyrisMessage
+from app.domain.chat.exercise_chat.exercise_chat_pipeline_execution_dto import (
+    ExerciseChatPipelineExecutionDTO,
+)
+from langchain_core.output_parsers import StrOutputParser
```

Ruff (0.8.2)

`langchain_core.output_parsers.StrOutputParser` imported but unused

Remove unused import: `langchain_core.output_parsers.StrOutputParser`

(F401)